container-images/assign-class-label/Dockerfile

```
@@ -0,0 +1,13 @@
+FROM python:3.9-slim
```

Is there a specific reason you're using an older Python? If not, I think you should use the current release.

```suggestion
FROM python:3.12-slim
```